TK-15920 Add details about accounts used with Windows Authentication (#477)
OutSystems/docs-product: src/setup-infra-platform/setup/default-configurations.md

|Log|LogServer.exe|LogServer.exe.config|Local System account|
|Scheduler Service|Scheduler.exe|Scheduler.exe.config|NT Service\OutSystems Scheduler Service|

> Note: When using SQL Server as the platform database with Windows Authentication, the OutSystems Deployment Controller Service and OutSystems Deployment Service will run under the Windows account configured as Admin in Configuration Tool, and the OutSystems Scheduler Service will run under the one configured as Runtime.

Reviewer: ```suggestion
Note: When using SQL Server as the platform database with Windows Authentication, the OutSystems Deployment Controller Service and OutSystems Deployment Service runs under the Windows account configured as Admin in Configuration Tool, and the OutSystems Scheduler Service runs under the one configured as Runtime.
```

> The Windows account configured as Admin in Configuration Tool will be granted local administrator permission (to deploy new applications and manage the IIS).

Reviewer: ```suggestion
The Windows account configured as Admin in Configuration Tool is granted local administrator permission (to deploy new applications and manage the IIS).
```
